src/App.js

```diff
@@ -47,16 +67,28 @@ class App extends Component {
     }, 1000);
   }
 
+  changeTheme = () => {
+    const newTheme = this.state.theme === 'primary' ? 'danger' : 'primary';
+    this.setState({theme:newTheme})
+  }
+
   render() {

```

Comment:
Please use: 
const { theme, motorcycles } = this.state 
just after render() {
And then in template use can use theme and motorcycles consts.
Using this.state.theme is quite repetitive.